hs_core/hydroshare/users.py

```diff
@@ -310,6 +179,7 @@ def list_users(query=None, status=None, start=None, count=None):
     return qs
 
 
+# Pabitra: This one seems to be not used anywhere. Can I delete it?
```

Comment:
@pkdash Same comment as above. Is there ever a need to list all groups from REST API? Looping in @alvacouch to get his input as well.